src/UberStrok.WebServices/ServerManager.cs

```diff
@@ -64,5 +67,19 @@ public ServerManager(WebServiceContext ctx)
         private readonly List<PhotonView> _gameServers;
 
         private readonly WebServiceContext _ctx;
+
+        public static string GetIPAddress(string hostname)
+        {
+            IPHostEntry host = Dns.GetHostEntry(hostname);
+
+            foreach (IPAddress ip in host.AddressList)
+            {
+                if (ip.AddressFamily == System.Net.Sockets.AddressFamily.InterNetwork)
+                {
+                    return ip.ToString();
+                }
+            }
+            return string.Empty;

```

Comment:
It would be best to throw an exception here, instead of silently returning an empty string.
```csharp
throw new Exception($"Unable to get host entry at {hostname}");
```